index.html

```diff
@@ -905,98 +757,380 @@ <h2>Test Vectors</h2>
-      [[JCS]].
-    </p>
+        <p>
+          With the following <code>inputOptions</code> provided to the signature suite along with the
+          <code>testNestedDocument</code> input document:
+        </p>
 
-    <p class="note">
-      This specification relies on EIP712, please review
-      [[EIP712]].
-    </p>
+        <pre class="example">
+          {
+            "embedAsURI": true,
+            "date": "2021-08-30T13:28:02Z",
+            "verificationMethod": "did:pkh:eip155:1:0xAED7EA8035eEc47E657B34eF5D020c7005487443#blockchainAccountId",
+            "domain": {
+              "name": "Test"
+            }
+          }
+        </pre>
 
-    <div class="issue">
-      TODO: We need to add a complete list of security considerations, e.g.,
-      what happens if EIP712 JSON schema does not match the message to be signed.
-    </div>
+        <p>The following is the resulting <code>proof</code> object:</p>
 
-  </section>
+        <pre class="example">
+          {
+            "created": "2021-08-30T13:28:02Z",
+            "proofPurpose": "assertionMethod",
+            "type": "EthereumEip712Signature2021",
+            "verificationMethod": "did:pkh:eip155:1:0xAED7EA8035eEc47E657B34eF5D020c7005487443#blockchainAccountId",
+            "proofValue": "0x8327ad5e4b2426eac7626400c75f000c3e04caf2a863b888988e4e85533880183d4b9cc6870183e55dabfa96b9486624f45ef849bb146257d123f297a2dbf3a11c",
+            "eip712": {
+              "domain": {
+                "name": "Test"
+              },
+              "types": "https://example.org/types.json",
+              "primaryType": "Document"
+            }
+          }
+        </pre>
+
+        <p>Dereferencing the <code>types</code> URI should result in the following object:</p>
+
+        <pre class="example">
+          {
+            "Data": [
+              {
+                "name": "job",
+                "type": "Job"
+              },
+              {
+                "name": "name",
+                "type": "Name"
+              }
+            ],
+            "Job": [
+              {
+                "name": "employer",
+                "type": "string"
+              },
+              {
+                "name": "jobTitle",
+                "type": "string"
+              }
+            ],
+            "Name": [
+              {
+                "name": "firstName",
+                "type": "string"
+              },
+              {
+                "name": "lastName",
+                "type": "string"
+              }
+            ],
+            "Document": [
+              {
+                "name": "@context",
+                "type": "string[]"
+              },
+              {
+                "name": "@type",
+                "type": "string"
+              },
+              {
+                "name": "data",
+                "type": "Data"
+              },
+              {
+                "name": "telephone",
+                "type": "string"
+              }
```

Comment:
Update to match the changes in 28bd5edecde8395242aea8ba64e9be25f59585d0.
With this change I can verify the test vector in "3.6.3 Nested Document - Types Generation - TypedData Schema as URI" in 28bd5edecde8395242aea8ba64e9be25f59585d0.
```suggestion
              {
                "name": "proof",
                "type": "Proof"
              },
              {
                "name": "telephone",
                "type": "string"
              }
            ],
            "Proof": [
              {
                "name": "created",
                "type": "string"
              },
              {
                "name": "proofPurpose",
                "type": "string"
              },
              {
                "name": "type",
                "type": "string"
              },
              {
                "name": "verificationMethod",
                "type": "string"
              }
```

Reply:
This change is committed in c7a3541509772818ff56c385d75cc57b778661de (#43).